posts/a-short-history-of-code-search.md

```diff
@@ -55,7 +55,7 @@ Also built on Apache Lucene is [OpenGrok](https://oracle.github.io/opengrok/), w
 
 There are many code search efforts today in active development that gesture toward what developers may expect from code search in the future. Some of these include closed source tools like Google’s internal search tool, making its large code base more accessible to those working on it. The work on Google’s code search began with gsearch by Jeff Dean in the early 2000s, which offered a productivity boost for developers and made large-scale refactoring easier. In 2010, Steve Yegge led the development of Grok at Google for code comprehension; a version of Grok was eventually open-sourced as Kythe in 2015 and is actively under development. Google Code Search, led by Russ Cox, was a free beta public product that debuted in Google Labs in 2006 but was shut down in 2011. There currently is a [publicly re-released](https://cs.opensource.google/) version that debuted in 2020 for use exclusively with Google open source software and Google Cloud Source Repositories.
 
-Founded in 2013 by Quinn Slack and Beyang Liu, Sourcegraph is open core software that offers universal code search across large code bases. Currently used by many enterprise companies as on-premises software, Sourcegraph is also available as [Sourcegraph cloud](https://sourcegraph.com), which enables developers to search across many publicly indexed open source repositories. Sourcegraph’s [long-term mission](https://handbook.sourcegraph.com/strategy-goals/strategy) is to help more people code, and, as we have seen with indexing books, being able to search across code can help more people understand others’ code and write their own code more effectively. 
+Founded in 2013 by Quinn Slack and Beyang Liu, Sourcegraph is open core software that offers universal code search across large code bases. Currently used by many enterprise companies as on-premises software, Sourcegraph is available at [Sourcegraph.com](https://sourcegraph.com). This tool enables developers to search across many publicly indexed open source repositories. Sourcegraph’s [long-term mission](https://handbook.sourcegraph.com/strategy-goals/strategy) is to help more people code, and, as we have seen with indexing books, being able to search across code can help more people understand others’ code and write their own code more effectively. 
```

Comment:
Hey @iskyOS - There were a handful of more `Cloud` references around /Learn, I updated them to my best judgement. Can you give a final pass if any of the language changed in this PR should be altered?